src/main/java/uk/ac/cam/cl/dtg/segue/api/useralerts/UserAlertsWebSocket.java

```diff
@@ -121,6 +121,21 @@ public UserAlertsWebSocket(final UserAccountManager userManager,
   }
 
 
+  private static Callback getGenericCallback() {
```

Comment:
I don't think that what you've done here is wrong, but I'll mention this just in case you weren't aware: there is a built-in `Callback.NOOP` that you could use instead of this. I think that the only difference is that the one you've created logs a message on failure. Given that we weren't doing that before, it would also be acceptable to just use `Callback.NOOP` to keep it simple.

But you can also leave it as it is, it's up to you, I don't think there is a clear winner here.